### PR TITLE
4.1/tactical version of failing->degraded switch (we'll bump openshif…

### PR DESCRIPTION
**Pull request by gabemontero**
…t/api in 4.2)

/assign @adambkaplan 
@openshift/sig-developer-experience fyi

### CODE REVIEW

File: pkg/operatorstatus/operatorstatus.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
 
-	"github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
+	v1 "github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
```
Comment (gabemontero) [inline pkg/operatorstatus/operatorstatus.go]: golang 1.11 / visual studio did this, not me

**Comment by gabemontero** [comment]
aws vpc pain on e2e-aws-operator

/test e2e-aws-operator

**Comment by gabemontero** [comment]
@adambkaplan given your "Arch update" email, do you want to bump openshift/api deps in the 4.1 timeframe (i.e. immediately more or less) and pick up the new constant from there, or are you fine with this change and bumping all that in 4.2

**Comment by adambkaplan** [comment]
@gabemontero Let's do the bump and pin any k8s dependencies to 1.13. This will make it easier for us to lock down our deps once we reach GA.

Golang changes can come in 1.12.

**Pull request closed without merging**